packages/core/src/shared/telemetry/util.ts

```diff
@@ -90,6 +91,37 @@ export function convertLegacy(value: unknown): boolean {
     }
 }
 
+/**
+ * Returns an identifier that uniquely identifies a single application
+ * instance/window of a specific IDE. I.e if I have multiple VS Code
+ * windows open each one will have a unique session ID. This session ID
```

Comment:
what matters is the toolkit instance, nothing else. we should be clear about that.
```suggestion
 * instance/window of a specific IDE. If I have multiple Toolkit extension instances (multiple vscode windows),
 * each one must have a unique session ID. This session ID
```